notebooks/json_blocks/WET/BLOCK-T413.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #23.        "maxiter": {
</pre><p>I Just noticed that we have out configurations all using snake case except this one in all the scripts. Maybe we should change to be consistent?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/ts_aos_analysis/pull/30/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>